packages/esm-appointments-app/src/patient-chart/patient-appointments-overview.component.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import PatientAppointmentsBase from './patient-appointments-base.component';
```

Comment:
FWIW, I was't able to figure out where this component was used.  I copied it over as-is, but was unable to test.